docs/diagnostics/PX1036.md

```diff
@@ -0,0 +1,53 @@
+# PX1036
+This document describes the PX1036 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                 | Type  | Code Fix    | 
+| ------ | ------------------------------------------------- | ----- | ----------- | 
+| PX1036 | The DAC primary key should be named "PK". The class containing DAC foreign keys should be named "FK".  | Warning (ISV Level 3: Informational) | Unavailable | 
+
+## Diagnostic Description
+The diagnostic checks the correct naming of key field declarations in DACs. For a primary key, the name should be PK. For foreign keys, the name of the static class which contains the foreign key classes must be FK.
+The warning is shown only for DACs that have either PXCacheName or PXPrimaryGraph attribute.
+
+To fix the issue, change the name of the primary key class to PK, the name of the class that contains foreign key classes to FK.
```

Comment:
Currently only the check and code fix for PK is implemented although I'm not sure if it should be mentioned.
The code fix for PK changes the name of the primary key to PK. Imho it should be described